jobtastic/task.py

```diff
@@ -157,7 +160,19 @@ def _get_possible_broker_errors_tuple(self):
             # Celery 2.5 uses `broker_connection` instead
             dummy_conn = self.app.broker_connection()
 
-        return dummy_conn.connection_errors + dummy_conn.channel_errors
+        possible_broker_errors = (
+            dummy_conn.connection_errors + dummy_conn.channel_errors
+        )
+        try:
+            from kombu.exceptions import OperationalError
+            # In celery 4.x when a broker is not connected it throws an
+            # OperationalError from kombu. It should also be noted that the
+            # newest version of kombu actually hangs forever. So we need to peg
+            # versions of kombu until that gets fixed.
```

Comment:
where is the kombu version peg happening?

Reply:
In tox.

Reply:
shoudnt it be done in requirements too?

Reply:
the point being that it would save some headache if somebody goes on the newer version of kombu and then installs jobtastic...

Reply:
i know kombu is not actually a requirement of jobtastic... hmm not sure what the best approach is here.

maybe it should be added to readme/changelog

Reply:
or maybe im not understanding things right. is the problem only in tests or would it actually cause an app to hang?

Reply:
I added a note in the readme.